_posts/elements/2015-07-01-grid.md

```diff
@@ -0,0 +1,878 @@
+---
+layout:       element
+title:        Grid System
+date:         2015-07-01 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- grid
+
+slug:         grid
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/pattern-library/sass/components/_grid.scss"
+
+description:  Grid system and settings that can be used throughout layouts.
+
+info: A responsive, mobile first fluid grid system, based on Susy and Breakpoint, that appropriately scales up to 12 columns as the device or viewport size increases. It includes some predefined classes for easy layout options, while providing Sass utilities for generating more semantic layouts.
+
+example: grid.html
+---
+
+<h3 class="hd-6 example-set-hd">Basic Grid</h3>
+<div class="example-set">
+    <div class="example-grid example-grid-basic">
+        <div class="grid-container">
+            <div class="col first">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+               <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col last">
+                <span class="example-label">Col</span>
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Basic Grid (with column shifts)</h3>
+<div class="example-set">
```

Comment:
Can you add a short intro for each grid on the PL pages so you have context as to what you’re looking at/what it does without jumping to the full external doc? example for this one: "The Basic Grid with Column Shifts allows you to create a two-column layout where there is (column-sized) space before or after the content." (who wrote that harrible trash!) :stuck_out_tongue_winking_eye: